solr/core/src/java/org/apache/solr/cloud/ZkDistributedQueue.java

```diff
@@ -51,9 +51,16 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * A ZK-based distributed queue. Optimized for single-consumer,
+ * <p>A ZK-based distributed queue. Optimized for single-consumer,
  * multiple-producer: if there are multiple consumers on the same ZK queue,
- * the results should be correct but inefficient
+ * the results should be correct but inefficient.</p>
+ *
+ * <p>This implementation (with help from subclass {@link OverseerTaskQueue}) is used for the
```

Comment:
Minor (and I know this could snowball): might be nice to add a note in OverseerTaskQueue. The doc for that class is pretty uninformative.

Reply:
There's already a link to this class from OverseerTaskQueue's javadoc. Leaving it as is.